Readme.rst

```diff
@@ -1,186 +1,186 @@
-    causes such combination to be necessarily infringed. The patent license
-    shall not apply to any other combinations which include the
-    Contribution.
-
-Except as expressly stated above, no rights or licenses from any copyright
-holder or contributor is granted under this license, whether expressly, by
-implication, estoppel or otherwise.
-
-DISCLAIMER
-
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
-LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
-CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
-SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
-INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
-CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
-ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
-POSSIBILITY OF SUCH DAMAGE.
-
-.. ===================================================================
-.. This is a bunch of directives to make the README file more readable
-.. ===================================================================
-
-.. CoreCI
-
-.. _Windows_VS2019: https://dev.azure.com/projectmu/mu/_build/latest?definitionId=53&&branchName=release%2F202405
-.. |WindowsCiBuild| image:: https://dev.azure.com/projectmu/mu/_apis/build/status/CI/Mu%20Silicon%20Intel%20Tiano%20CI%20VS2019?branchName=release%2F202405
-.. |WindowsCiTest| image:: https://img.shields.io/azure-devops/tests/projectmu/mu/53.svg
-.. |WindowsCiCoverage| image:: https://img.shields.io/badge/coverage-coming_soon-blue
-
-.. _Ubuntu_GCC5: https://dev.azure.com/projectmu/mu/_build/latest?definitionId=54&branchName=release%2F202405
-.. |UbuntuCiBuild| image:: https://dev.azure.com/projectmu/mu/_apis/build/status/CI/Mu%20Silicon%20Intel%20Tiano%20CI%20Ubuntu%20GCC5?branchName=release%2F202405
-.. |UbuntuCiTest| image:: https://img.shields.io/azure-devops/tests/projectmu/mu/54.svg
+=========================================
+Project Mu Silicon Intel Tiano Repository
+=========================================
+
+============================= ================= =============== ===================
+ Host Type & Toolchain        Build Status      Test Status     Code Coverage
+============================= ================= =============== ===================
+Windows_VS2022_               |WindowsCiBuild|  |WindowsCiTest| |WindowsCiCoverage|
+Ubuntu_GCC5_                  |UbuntuCiBuild|   |UbuntuCiTest|  |UbuntuCiCoverage|
+============================= ================= =============== ===================
+
+This repository is part of Project Mu.  Please see Project Mu for details https://microsoft.github.io/mu
+
```

Comment:
Was this a line ending change missed in the earlier commit? Or is it converting to LF?

Reply:
I rearranged commits and this replayed the LF file with the change on top of the now CRLF file. I will reorder it with just the delta needed.

Reply:
Done